src/azure/ApiCenterDefines/ApiCenterApi.ts

```diff
@@ -60,8 +60,11 @@ export class ApiCenterApisDataplane implements IApiCenterApisBase {
     async getChild(context: ISubscriptionContext, content: string): Promise<GeneralApiCenterApi[]> {
         let server = new ApiCenterDataPlaneService(context);
         const res = await server.getApiCenterApis();
-        this._nextLink = res.nextLink;
-        return res.value;
+        if (res) {
```

Comment:
Just corious, will control plane have similar issue? Or in control plane API, it is return `[]`, while data plane API, it is return `undefined`?

Reply:
no, the control plane list the apis from graph call. which needs user sign in, then siwtch tenant, select subs, then graph will query all the apis in this sub. so all the apis must be existed.
but data plane not, data plane import this apis by user input, in this senario, the rest api will return undefined.